### PR TITLE
Removed space within parsed VPNBOOK password.

### PR DESCRIPTION
**Pull request by colinwebber**
VPNBOOK password is provided as an image. api.ocr.space/parse/image API erroneously sees a space between certain character combinations. Since passwords generally do not contain spaces, this extra awk expression removes the space.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ workflows:
 jobs:
   build-ubuntu-image:
     environment:
-      IMAGE_NAME: haugene/transmission-openvpn
+      IMAGE_NAME: colinwebber/transmission-openvpn
```
Comment (pkishino) [inline .circleci/config.yml:56]: this file should not be part of the commit as this will link to the wrong image

Comment (pkishino) [reply .circleci/config.yml:56]: Ah, so , if you remove the commit b97d817 then it will be fine

Comment (colinwebber) [reply .circleci/config.yml:56]: Apologies, I did that for testing purposes but committed to the same branch. I'm going to create a new pull request to avoid any remnants.
